commands/admin.js

```diff
@@ -115,20 +126,20 @@ exports.promote = function(message, args) {
     }
     var id_rx = /^<@([0-9]+)>$/g;
     var id = id_rx.exec(args[0]);
-    console.log(id);
-    id = id ? id[1] : args[0]
-         console.log(id);
+    id = id ? id[1] : args[0];
     var user = global.util.getUser({id: id});
+    logger.info(`Locating user ${id}`);
     if(user === undefined || global.userList[user.id]>=2){
-        message.channel.send(`Cannot promote that user!`)
+        message.channel.send(`Cannot promote that user!`);

```

Comment:
Can we modify this error message to be more specific on the undefined failure vs permissions capped